content/release-10-14-0/announcements-10-14-0.md

```diff
@@ -23,13 +23,14 @@ As of release 10.16+, new tenants can utilize a new enhanced time-series support
 The following APIs will be removed without any replacement:
 
 * `GET /measurement/measurements/{id}`
-* `DEL /measurements/measurement/getById`
+* `DEL /measurements/measurement/{id}`
 
-The following API will no longer be supported:
+The following API is partially supported:
 
 * `DEL /measurements/measurement/`
 
-Instead, a time-to-live configuration or retention rules can be used to remove expired measurements data from the Operational Store.
+In 10.16 The parameters `dateFrom` and `dateTo` are not supported. In 10.17+ the parameters `dateFrom` and `dateTo` must be truncated to full hours (for example, 2022-08-19T14:00:00.000Z), otherwise an error will be returned.
```

Comment:
```suggestion
In release 10.16, the parameters `dateFrom` and `dateTo` are not supported. In 10.17+ the parameters `dateFrom` and `dateTo` must be truncated to full hours (for example, 2022-08-19T14:00:00.000Z), otherwise an error will be returned.
```